autoload/EasyMotion/cmigemo.vim

```diff
@@ -92,7 +92,13 @@ function! EasyMotion#cmigemo#getMigemoPattern(input) "{{{
 
     if has('migemo')
         " Use migemo().
-        return migemo(a:input)
```

Comment:
if the users set 'migemodict', easymotion should respect the option.

I think we should check &migemodict is empty or not before overwriting the value.